src/modules/student/dto/create-student-profile.dto.ts

```diff
@@ -16,17 +11,9 @@ import {
   MaxLength,
   Max,
   IsUrl,
-  IsEmail,
 } from 'class-validator';
 
 export class CreateStudentProfileDto implements NewStudentProfileInterface {
-  @IsString()
-  id: string;
-
-  @IsNotEmpty()
-  @IsEmail()
-  initialData: StudentInitialInterface;
```

Comment:
initialData musi być w DTO, bo to jest wymagane aby profil istniał. Mój zamysł był taki żeby właśnie od początku był związany z danymi inicjacyjnymi dlatego myślę, że to pole powinno być obowiązkowe.
Będzie tu podawany (lub ściągany z danych usera) adres email.